packages/gestalt/src/FlyoutUtils/Contents.js

```diff
@@ -379,8 +379,17 @@ export default class Contents extends React.Component<Props, State> {
     const windowSize = {
       height: window.innerHeight,
       width: window.innerWidth,
-      scrollY: positionRelativeToAnchor ? 0 : window.scrollY, // scroll not needed for relative elements
-      scrollX: positionRelativeToAnchor ? 0 : window.scrollX,
+      // scroll not needed for relative elements
+      scrollY: positionRelativeToAnchor
+        ? 0
+        : window.pageYOffset ||
+          (document.documentElement && document.documentElement.scrollTop) ||
```

Comment:
Can you include a comment here (or extract as a named variable) as to why we have to do this so that it doesn't get accidentally removed in the future?